### PR TITLE
[13.0][FIX] base_location_thailand

### PR DESCRIPTION
**Pull request by ps-tubtim**
Issue: https://github.com/OCA/l10n-thailand/issues/106, https://github.com/OCA/l10n-thailand/issues/109

### CODE REVIEW

File: base_location_thailand/wizard/geonames_import.py [hunk 1]
```diff
@@ -30,3 +30,7 @@ def get_and_parse_csv(self):
             data_file.close()
             return parsed_csv
         return super().get_and_parse_csv()
+
+    @api.depends("country_id")
+    def _compute_is_thailand(self):
+        self.is_thailand = self.country_id.code == "TH"
```
Comment (Saran440) [inline base_location_thailand/wizard/geonames_import.py]: please, add `self.ensure_one()`

Comment (ps-tubtim) [reply base_location_thailand/wizard/geonames_import.py]: @Saran440 Thanks for review. Edited.

**Review by Saran440 — APPROVED** [review]
Code reviewed. LGTM.

File: base_location_thailand/wizard/geonames_import.py [hunk 2]
```diff
@@ -10,14 +10,19 @@
 class CityZipGeonamesImport(models.TransientModel):
     _inherit = "city.zip.geonames.import"
 
-    country_name = fields.Char(related="country_id.name", readonly=True)
+    is_thailand = fields.Boolean(compute="_compute_is_thailand", readonly=True)
     location_thailand_language = fields.Selection(
```
Comment (kittiu) [inline base_location_thailand/wizard/geonames_import.py:20]: Please add help,
```
For Thailand only, data is from TH_th.txt and TH_en.txt stored in the module's data folder. To get data from Geonames.org, please uninstall base_location_thailand.
```

**Comment by pedrobaeza** [comment]
Shouldn't this be called `l10n_th_base_location`?

**Comment by kittiu** [comment]
Agreed, thank you.

File: base_location_thailand/wizard/geonames_import.py (hunk 2 above)
Comment (ps-tubtim) [reply base_location_thailand/wizard/geonames_import.py:20]: Added.

**Pull request merged**